zk_dtypes/_ecinfo.py

```
@@ -145,6 +230,91 @@ def __init__(self, ec_type):
       self.point_repr = 'xyzz'
       self.curve_group = 'g1'
       self.is_montgomery = True
+    # Pasta G1 types (non-pairing, base field is the curve's scalar field dtype)
+    elif ec_type == _pallas_g1_affine_dtype:
+      self.base_field_dtype = _pallas_sf_dtype
+      self.storage_bits = 512  # 2 × 256 bits (x, y)
```

Ruff (0.15.17)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)